ricoconverter/ricoconverter-convert/src/test/resources/eac2rico/_37emptyDates/expected.xml

```diff
@@ -34,7 +34,7 @@
          </rico:AgentName>
       </rico:hasOrHadAgentName>
       <rico:agentIsSourceOfAgentHierarchicalRelation rdf:resource="agentHierarchicalRelation/005205-111111-19470101-*"/>
-      <rico:hasOrHadSubordinate rdf:resource="agent/111111" />
+      <rico:hadSubordinate rdf:resource="agent/111111" />
```

Comment:
It should be rico:hasDirectSubordinate here (in the input file there is no end date for the relation, so we consider it is current).